radio/src/gui/128x64/view_statistics.cpp

```diff
@@ -140,30 +125,18 @@ void menuStatisticsDebug(event_t event)
       break;
 
     case EVT_KEY_FIRST(KEY_UP):
-#if defined(KEYS_GPIO_REG_PAGEDN)
     case EVT_KEY_BREAK(KEY_PAGEDN):
       disableVBatBridge();
       chainMenu(menuStatisticsDebug2);
       break;
-#elif defined(NAVIGATION_X7)
-    case EVT_KEY_BREAK(KEY_PAGE):
-      disableVBatBridge();
-      chainMenu(menuStatisticsDebug2);
-      break;
-#endif
 
     case EVT_KEY_FIRST(KEY_DOWN):
-#if defined(KEYS_GPIO_REG_PAGEUP)
     case EVT_KEY_BREAK(KEY_PAGEUP):
-#elif defined(NAVIGATION_X7)
-    case EVT_KEY_LONG(KEY_PAGE):
-#endif
-      killEvents(event);
       disableVBatBridge();
```

Comment:
Not sure why this should be needed in the UI code, as I believe it is done automatically in the ADC code.